arches_lingo/views/api/concepts.py

```diff
@@ -91,3 +94,139 @@ def default_sensitivity():
         if elastic_prefix_length >= 5:
             return 0
         return int(5 - elastic_prefix_length)
+
+
+@method_decorator(
+    group_required("RDM Administrator", raise_exception=True), name="dispatch"
+)
+class ConceptResourceView(ConceptTreeView):
+    def get(self, request):
+        scheme = request.GET.get("scheme", None)
+        exclude = request.GET.get("exclude", None)
+        term = request.GET.get("term", None)
+        page_number = request.GET.get("page", 1)
+        items_per_page = request.GET.get("items", 25)
+        concepts = request.GET.get("concepts", None)
+        concept_ids = concepts.split(",") if concepts else None
+        Concept = SemanticResource.as_model("concept")
+
+        if not concept_ids:
+            # if scheme:
+            #     if exclude == "true":
+            #         concept_query = Concept.exclude(
+            #             part_of_scheme__0__0__resourceId=scheme
```

Comment:
part_of_scheme is now cardinality 1, I wonder if that's what changed here. In any event, the syntax [here](https://github.com/archesproject/arches-querysets/pull/18) is now working for this example, so I'll merge it soon to unblock work here.